lib/gui/app/components/flash-results/flash-results.tsx

```diff
@@ -162,9 +163,10 @@ export function FlashResults({
 					<Txt>{middleEllipsis(image, 24)}</Txt>
 				</Flex>
 				<Txt fontSize={24} color="#fff" mb="17px">
-					Flash {allFailed ? 'Failed' : 'Complete'}!
+					{i18next.t('flash.flash')}{' '}
+					{allFailed ? i18next.t('failed') : i18next.t('completed')}!
```

Comment:
Probably better to do this by choosing from two "complete strings" e.g.
```typescript
{allFailed ? i18next.t('flash.failed') : i18next.t('flash.completed')}
```
(where `flash.failed` would be "Flash Failed!"), rather than assuming that the separate strings can be joined by a space character, and in a particular order? (There might be some languages where the 'failed' text needs to come before the 'flash' text. And some languages might not use the exclamation-mark in the same way.)

Reply:
Finished, thanks.